src/zfstools/models.py

```diff
@@ -139,10 +147,14 @@ def lookup(self, name):
 
         return dset
 
-    def parse_zfs_r_output(self, creationtimes):
+    def parse_zfs_r_output(self, creationtimes, properties):
+        def extract_properties( line ):
+            items = s.strip().split( '\t' )
```

Comment:
Actually, the length of the list expected from this split is len(properties) + 1 (for the 'name' property).  Added an assert to clarify and verify this.